s3-sink-connector/src/main/java/io/aiven/kafka/connect/s3/config/S3SinkConfig.java

```diff
@@ -153,12 +83,13 @@ static Map<String, String> preprocessProperties(final Map<String, String> proper
     }
 
     private static Map<String, String> handleDeprecatedYyyyUppercase(final Map<String, String> properties) {
```

Comment:
can we totally reuse this method from S3CommonConfig ? Its duplicated there too.